service/dap/types.go

```diff
@@ -252,6 +254,11 @@ type AttachConfig struct {
 	// Wait for a process with a name beginning with this prefix.
 	AttachWaitFor string `json:"waitFor,omitempty"`
 
+	// GuessSubstitutePath is used to automatically guess SubstitutePath if it
+	// is not specified explicitly. It should be copied from the output of
+	// 'dlv substitute-path-guess-helper'.
+	GuessSubstitutePath *api.GuessSubstitutePathIn
```

Comment:
(optional) to be consistent with other fields style, it may be `json:"guessSubstitutePath,omitempty"`.

I'm ok with keeping the json encoding of  `api.GuessSubstitutePathIn` fields as they are though because that's an output of `dlv substitute-path-guess-helper` output.

Reply:
Done.